src/Groups/homomorphisms.jl

```diff
@@ -285,6 +288,50 @@ function isisomorphic(G::GAPGroup, H::GAPGroup)
   end
 end
 
+"""
+    isomorphic_perm_group(G::GAPGroup)
+Return a group of permutation `H` and an isomorphism `f` from `G` to `H`.
+"""
+function isomorphic_perm_group(G::GAPGroup)
+   f = GAP.Globals.IsomorphismPermGroup(G.X)
+   if f==GAP.Globals.fail
+      throw(ArgumentError("Could not convert group into a permutation group"))
+   else
+      H = GAP.Globals.Image(f)
+      n = GAP.Globals.NrMovedPoints(H)
+      H = PermGroup(H,n)
+      return H, _hom_from_gap_map(G,H,f)
+   end
+end
+
+"""
+    isomorphic_pc_group(G::GAPGroup)
+Return a polycyclic group `H` and an isomorphism `f` from `G` to `H`.
```

Comment:
```suggestion
Return a pc group `H` and an isomorphism `f` from `G` to `H`.
If `G` is infinite or not solvable then no such isomorphism exists and an exception is thrown.
```

Every pc group is a polycyclic group, but not every polycyclic group is a pc group! Being polycyclic is a mathematical property, and e.g. a permutation group can satisfy it; being a "pc group" is a matter of how the group is represented internally.